core/src/com/ericc/the/game/systems/realtime/RenderSystem.java

```diff
@@ -163,23 +166,23 @@ This complicated algorithm attempts to paint convincing walls with a tileset rep
         if ((code & 0b000010000) != 0) {
             // Floor tile.
             batch.draw(Media.getRandomFloorTile(
-                    x, y, CurrentMap.map.getRandomNumber(x, y, TileTextureIndicator.FLOOR.getValue()), isStatic
-                    ), x, y, 1, 1);
+                    x, y, map.getRandomNumber(x, y, TileTextureIndicator.FLOOR.getValue()), isStatic
+            ), x, y, 1.001f, 1.001f);
```

Comment:
czemu taka zmiana?

Reply:
+1

Reply:
Czasami pomiędzy tilami pojawiały się jednopikselowe linie przerwy z powodu jakichś zaokrągleń floatowych. Szczególnie dobrze widać to na nagraniach. Po tej zmianie narysowane tile zachodzą trochę na siebie i glitche znikły, przynajmniej u mnie.